app/controllers/weets.js

```diff
@@ -0,0 +1,6 @@
+const { getRandomWeet } = require('../services/weetService');
```

Comment:
in this case the controller is not necessary due to the acceptance criteria of the user history